db/seeds.rb

```diff
@@ -1,7 +1,13 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+DynamicFieldCategory.destroy_all
+DynamicFieldGroup.destroy_all
+DynamicField.destroy_all
+
+user = User.find(1);
+
+DynamicFieldCategory.create!({id: 1, display_label: "Descriptive Metadata", sort_order: 1})
+
+df_group = DynamicFieldGroup.create!({id: 1, string_key: 'title', display_label: 'Title', sort_order: 0, is_repeatable: 0, parent_id: 1, created_by: user, updated_by: user, parent_type: "DynamicFieldCategory", xml_translation: '{\n    \"element\": \"mods:titleInfo\",\n    \"content\": [\n        {\n            \"element\": \"mods:nonSort\",\n            \"content\": \"{{title_non_sort_portion}}\"\n        },\n        {\n            \"element\": \"mods:title\",\n            \"content\": \"{{title_sort_portion}}\"\n        }\n    ]\n}'})
```

Comment:
I would stray away from setting the identifiers as part of this process because if the developer has already created some dynamic fields, groups or categories with the same ids they would get errors. I think you should be able to set the category for the DynamicFieldGroup by using the `parent` field and passing in the DynamicFieldCategory created above, that way you don't have to reference the identifiers.

Reply:
Agreed!  We can rely on database auto-increment id for this.